cves/CVE-2016-7905.yml

```diff
@@ -334,8 +361,12 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: |
+      The vulnerable function takes any inputs. When the value doesn’t exist,
```

Comment:
Complex inputs is the idea that a systems vulnerability can be reduced and mitigated by reducing the complexity of the input. It does not mean that a specific file types complexity is inherently a bad thing but rather reducing the users ability to have too much input in the system.